test/v2/unit/recovery.js

```diff
@@ -46,7 +46,8 @@ describe('Recovery:', function() {
         bitgoKey: 'xpub661MyMwAqRbcGsSbYgWmr9G1dFgPE8HEb1ASRShbw9S1Mmu1dTQ7QStNwpaYFESq3MeKivGidN8twMeJzqh1veuSP1t2XLENL3mwpatfTst',
         walletPassphrase: TestV2BitGo.V2.TEST_WALLET1_PASSCODE,
         recoveryDestination: '2NB5Ynem6iNvA6GBLZwRxwid3Kui33729Nw',
-        scan: 5
+        scan: 5,
+        ignoreSegwit: true
```

Comment:
I've added ignoreSegwit:true here because this test case does not have segwit unspents. It's also easier than manually nocking 10 more api routes for every coin in this test suite.